content/en/user-guide/localstack-enterprise/k8s-operator/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "K8s Operator"
+linkTitle: "K8s Operator"
+weight: 2
+description: Custom K8s operator that offers LocalStack emulator as a native resource in Kubernetes environments.
+tags: ["Enterprise plan"]
+---
+
+## Introduction
+
+LocalStack K8s operator is a custom Kubernetes operator that offers LocalStack emulator as a native resource in Kubernetes environments.
+The operator is designed to simplify the deployment and management of LocalStack in Kubernetes clusters.
+
+## Installation
+
+To install the K8s operator, run the following commands:
+
+{{< command >}}
+$ kubectl apply -f https://raw.githubusercontent.com/localstack/localstack-k8s-operator/v0.2.0/release/crds.yaml
+$ kubectl apply -f https://raw.githubusercontent.com/localstack/localstack-k8s-operator/v0.2.0/release/controller.yaml
+{{< / command >}}
+
+You can then deploy a LocalStack in the following manner:
```

Comment:
nit: Maybe we could be more specific here and say something like: 
> You can then deploy a LocalStack instance by storing the following file content as `localstack.yml` and applying it against the cluster via `kubectl apply -f localstack.yml`. 

I guess that might be clear for most readers, but not for all. 🤷